File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/org/openapitools/model/UsernamePassword.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param username The Mojang account e-mail or username. Never store it.
 * @param password The Mojang account password, never store it.
 */
data class UsernamePassword (

        @get:NotNull 
        @JsonProperty("username") val username: String,

        @get:NotNull 
        @JsonProperty("password") val password: String
) {

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/org/openapitools/client/model/UsernamePassword.scala

package org.openapitools.client.model


case class UsernamePassword (
    /* The Mojang account e-mail or username. Never store it. */
    _username: String,
    /* The Mojang account password, never store it. */
    _password: String
)
object UsernamePassword {
    def toStringBody(var_username: Object, var_password: Object) =
        s"""
        | {
        | "username":$var_username,"password":$var_password
        | }
        """.stripMargin
}
